requirements.txt

```diff
@@ -1,4 +1,4 @@
-cirq~=1.0.0
-cirq-superstaq~=0.3.16
+cirq~=1.2.0.dev2022
```

Comment:
does supermarq actually need the pre-release version of cirq? if not we should get rid of this requirement so it just uses the cirq version required by cirq-superstaq

Reply:
Imo we should opt to have loose requirements in public-facing repos.  For example [`css` only requires `cirq>=0.14.0`](https://github.com/SupertechLabs/cirq-superstaq/blob/main/requirements.txt).  It's fine to update to, say, `>=1.0.0` or `1.2.0` if we rely on features therein, but I agree with @richrines1 and think that pre-releases should be avoided.